src/MPAGSCipher/TransformChar.cpp

```diff
@@ -0,0 +1,49 @@
+#include "TransformChar.hpp"
+
+
+std::string transformChar( const char in_char )
+{
+    std::string inputText{""};
+
+    // Uppercase alphabetic characters
+    if (std::isalpha(in_char)) {
+        inputText += std::toupper(in_char);
```

Comment:
Can return early here to improve efficiency:
```suggestion
        inputText += std::toupper(in_char);
        return inputText;
```